runtime/api/init.js

```diff
@@ -117,6 +117,7 @@ function createInstanceContext (id, options = {}, data) {
   Object.assign(runtimeContext, services, {
     weex,
     getJSFMVersion,
+    requireModule: weex.requireModule,
```

Comment:
I think using `(..args) => weex.requireModule(...args)` is more reliable. We should not assume that the context of `requireModule` has bind to `weex` even if it actually does.

Reply:
done.